wallet/src/main/java/com/android/identity_credential/wallet/MainActivity.kt

```diff
@@ -128,6 +206,17 @@ class MainActivity : ComponentActivity() {
                             }
                         }
                     )
+                    NavigationDrawerItem(
+                        icon = { Icon(imageVector = Icons.Filled.Settings, contentDescription = null) },
+                        label = { Text(text = "Settings") },
+                        selected = false,
+                        onClick = {
+                            scope.launch {
+                                drawerState.close()
+                                navigation.navigate("TransferSettingsScreen")
+                            }
+                        }
+                    )
```

Comment:
We should not expose any settings related to data transfer, at least not initially. We should just support QR and NFC engagement with _mdoc central client_ mode and that's it. (By the same token, you don't see Google Wallet or Apple Wallet expose any such settings to their users.)